flocker/node/agents/test/test_blockdevice.py

```diff
@@ -441,10 +444,12 @@ def test_one_device(self):
         and the volume's filesystem is mounted in the right place.
         """
         dataset_id = uuid4()
+        requested_size = REALISTIC_BLOCKDEVICE_SIZE
```

Comment:
Given `allocation_unit()` I think `REALISTIC_BLOCKDEVICE_SIZE` should die, since it shouldn't have to try to support N backends, and be replaced with a dynamically calculated value based on `allocation_unit()`.